data/en/fileappend.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name":"fileAppend",
 	"type":"function",
-	"syntax":"fileAppend(filePath, data [, charset])",
+	"syntax":"fileAppend(file, data [, charset])",
```

Comment:
An additional optional parameter of `addNewLine` should also be added after `charset`.

Reply:
Thanks @shawnoden I'll make that change after I merge this one.